### PR TITLE
Reduce log output when no tests are prioritized

### PR DESCRIPTION
**Pull request by ZainRizvi**
### <samp>🤖 Generated by Copilot at 733b991</samp>

Improve test reordering output in `tools/testing/test_selections.py`. Add a check to only print reordering information when there are tests to prioritize.

### CODE REVIEW

**Review by malfet — APPROVED** [review]

File: tools/testing/test_selections.py [hunk 1]
```diff
@@ -241,16 +241,19 @@ def print_tests(tests: Set[str], test_group_description: str) -> None:
         return ([], tests)
 
     # TODO: Would be great to upload these stats to RDS/Rockset!
-    test_cnt_str = pluralize(len(tests), "test")
-    print(f"Reordering tests: Prioritizing {len(bring_to_front)} of {test_cnt_str}")
-    print(
-        f"Prioritized tests estimated to run up to {duration_to_str(time_savings_sec)} sooner than they would've otherwise"
-    )
+    if len(bring_to_front):
```
Comment (kit1980) [inline tools/testing/test_selections.py:244]: `if bring_to_front:` looks more idiomatic.

**Review by kit1980 — APPROVED** [review]

**Review by kit1980 — APPROVED** [review]

File: tools/testing/test_selections.py (hunk 1 above)
Comment (ZainRizvi) [reply tools/testing/test_selections.py]: true

**Pull request closed without merging**